src/lib/routes/admin-api/user-admin.ts

```diff
@@ -114,8 +122,14 @@ export default class UserAdminController extends Controller {
                 openApiService.validPath({
                     tags: ['Users'],
                     operationId: 'validateUserPassword',
+                    summary: 'Validate password for a user',
+                    description:
+                        'Validate the password strength. Minimum 10 characters, uppercase letter, number, special character.',
                     requestBody: createRequestSchema('passwordSchema'),
-                    responses: { 200: emptyResponse },
+                    responses: {
+                        200: emptyResponse,
+                        ...getStandardResponses(400, 401),
```

Comment:
Any user request that sends data can also get a 415 if the data is in a format we don't accept (such as xml). This also goes for the other endpoints in this pr.

```suggestion
                        ...getStandardResponses(400, 401, 415),
```